.github/workflows/test-emacs-plugin.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+name: test Emacs plugin
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    name: Emacs
+    runs-on: ubuntu-latest
+    container: ghcr.io/quick-lint/quick-lint-js-github-builder:v1
+    steps:
+      - name: checkout
+        if: ${{ github.event_name == 'push' }}
+        uses: actions/checkout@v2
+      - name: checkout
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: version
+        run: emacs --version
+
+      - name: test
+        run: ./plugin/emacs/test-emacs.sh
```

Comment:
This won't work because quick-lint-js isn't installed. You'll need to do something like .github/workflows/test-install.yml does to install quick-lint-js first.